src/pages/guides/using/marketo/marketo-data-streams.md

```diff
@@ -0,0 +1,78 @@
+# Subscribing to the Marketo Events with Adobe I/O Events
+
+These instructions describe how to set up and get started using Adobe I/O Events for the Marketo Data Streams.
+
+- [Introduction](#introduction)
+- [Subscribing to Marketo Events](#subscribing-to-marketo-events)
+- [Developer Guidelines](#developer-guidelines)
+- [Debug](#debug)
+
+## Introduction
+
+In order to allow for additional scalability and further programmatic automation of the Marketo platform, we have introduced the Marketo Data Streams.  The data streams are available to all CET and Performance Plus customers.  There are currently five data streams:
+
+- Lead Activity Data Stream
+- Metrics Data Stream (Alpha)
+- Notification Data Stream (Beta)
+- Observability Data Stream (Beta)
+- User Audit Data Stream (Beta)
```

Comment:
@ray-eckard I'd say let's make these links to the docs for each stream that we have docs for.  So Lead Activity Data Stream will link to the Marketo public docs we have for it (I'm working on getting the link), Metrics Data Stream I don't believe we have public docs for so there'd be no link, Notification data stream will eventually have a page in these IO Docs so this can be left unlinked until we create that, Observability and Audit streams can be linked to their IO Docs pages.  This would resolve @francoisledroff 's question below

Reply:
Can probably use this url for lead activity stream:
https://developers.marketo.com/data-streams/#overview_lead_activity_data_stream